salt/proxy/cimc.py

```diff
@@ -150,7 +150,15 @@ def set_config_modify(dn=None, inconfig=None, hierarchical=False):
                                 decode=True,
                                 verify_ssl=False,
                                 raise_error=True,
+                                status=True,
                                 headers=DETAILS['headers'])
+
+    if str(r['status']) not in ['200', '201', '202', '204']:
+        log.debug("Received error HTTP status code: {0}" .format(str(r['status'])))
+        logout(cookie)
+        raise salt.exceptions.CommandExecutionError(
+            "Did not receive a valid response from host.")
```

Comment:
Maybe log what the error code we received was?

Reply:
It should be logging the error code on line 157.